tests/test_model.py

```diff
@@ -10,7 +12,9 @@ class CrashSimilarityTest(unittest.TestCase):
     # Train Model to be used in all tests
     @classmethod
     def setUpClass(self):
-        self.paths = ['tests/test.json']
+        current_abs_path = os.path.abspath(os.path.dirname(__file__))
+        project_abs_path = Path(current_abs_path).parent
+        self.paths = [os.path.join(project_abs_path, 'tests/test.json')]
```

Comment:
Why is this change needed?

Reply:
I had a strange problem running the test suite locally as it couldn't find `test.json` file, so I had to do this change to fix it...I think we should introduce a better mechanism to deal with relative paths (test.json path, trained model paths), like a configuration file?

Reply:
Can you try removing these changes and seeing if the problem is still there? And copy here the logs so I can see it.

Reply:
I tried again locally and things are running normally :smiley:, ready to merge :heavy_check_mark: